po/rufus.pot

```diff
@@ -216,6 +216,10 @@ msgctxt "AdvancedOptionsText"
 msgid "For advanced install options using an <b>Endless USB Stick</b>, <a id='AdvancedOptionsLink'>click here</a>."
 msgstr ""
 
+msgctxt "PrivacyPolicyText"
+msgid "This installer uses Google Analytics to record anonymous data. <a id='PrivacyPolicyLink'>Click here</a> for more information, and to learn how to opt out."
```

Comment:
Can we provide a touch more rationale here, or do we run out of space? Eg: anonymous usage data to improve the installer for everybody.

Reply:
The wording in the FAQ is:

> The Endless Installer for Windows uses Google Analytics to record anonymous data that allows us to improve the experience for you and other users. These are the same types of analytics already built into the Endless website and covered by our privacy policy.

We could say:

> We collect anonymous data to improve this installer for everybody. Click here for more information, and to learn how to opt out.

?

Reply:
collect? record?

Reply:
LGTM - collect seems better in the abridged version, record seems OK when we're talking about using GA